tests/ESPresense.Companion.Simulation/AccuracyHarness/Csi/csi_replay.py

```
@@ -0,0 +1,287 @@
+
+# --------------------------------------------------------------------------- CLI
+def _cmd_replay(args):
+    rate = None
+    realtime = False
+    as_json = False
+    paths = []
+    i = 0
+    while i < len(args):
+        a = args[i]
+        if a == "--rate":
+            rate = int(args[i + 1]); i += 2; continue
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)